client/app/bundles/course/lesson_plan/containers/LessonPlan.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Router, Route, browserHistory } from 'react-router';
+import LessonPlanContainer from './LessonPlanContainer';
```

Comment:
Now there're `LessonPlanContainer` and `LessonPlan`  inside containers folder, perhaps they can be better named and structured ?